uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/manager-leave/components/manager-leave-requests.js

```diff
@@ -189,7 +189,7 @@ define([
      */
     vm.getUserNameByID = function (id) {
       var user = _.find(vm.filteredUsers, function (contact) {
-        return contact.contact_id === id;
+        return contact.id === id;
```

Comment:
`id` is used when using `contact` object, and `contact_id` is used only for the `LeaveRequest`filter. The PR description is confusing, will fix it.